identity/webapp/pages/api/auth/[...auth0].ts

```diff
@@ -7,22 +7,16 @@ export default auth0.handleAuth({
     if (error) {
       const query = new URLSearchParams(req.url);
       res.redirect(`/account/error?${query.toString()}`);
+      return;
     }
 
     // We have to `try … catch` here so we don't raise an Internal Server Error
     // when the Auth0 callback fails for explicable reasons, e.g. somebody sending
     // a deliberately malformed token or code.
-    //
-    // We could include the error message:
-    //
-    //      .end(error.message)
-    //
-    // but I haven't because I don't think anybody will encounter this in normal running,
-    // and I'm not sure if that message might leak sensitive info.
     try {
       return await auth0.handleCallback(req, res);
     } catch (error) {
-      res.status(error.status || 500);
+      res.status(error.status || 500).end(error.message);
```

Comment:
This isn't part of this PR, but I discovered it while working on it – if you don't send the message, the endpoint never returns.

Reply:
Good catch. As per the deleted comment though - this may expose errors that we don't really want to expose, could you `end` without `error.message`?

Reply:
Sure, I've dropped in a hard-coded message and a warning log.